tests/asr/test_process_noise_info.py

```diff
@@ -0,0 +1,16 @@
+from eevee.metrics.asr import extract_info_tags, remove_info_tags, define_noisy
+
+
+def test_process_noise_info():
+
+	clean_transcription = "This is an example"
+	noise_tags = ["", "<audio_silent>", "<inauidble>", "<background_speech>"]
+	noise_labels = [0, 0, 1, 1]
+
+	noisy_transcriptions = ["{}{}".format(clean_transcription, tag) for tag in noise_tags]
```

Comment:
you sure you don't want spaces `{}{}` vs `{} {}` to be part of the tagged transcriptions and test it as well?
im not sure how transcriptions get tagged/predicted with or without spaces.

Reply:
i have seen both spaces and no spaces in the (ops) annotations - between info tags and words

counter-question: do spaces matter when calculating WER ? do you strip away spaces or consider them as words/characters ?

Reply:
either way, this is not an issue. `remove_info_tags()` strips trailing and leading spaces [here](https://github.com/skit-ai/eevee/blob/eea47e8973226e258aa4a14b44146243c2ce8fce/eevee/metrics/asr.py#L673)

Reply:
@sachinkum can answer your question, on the best/correct way to do it with/without spaces.